log/src/main/java/kcl/teamIndexZero/traffic/log/Log_Config.java

```diff
@@ -1,6 +1,6 @@
 package kcl.teamIndexZero.traffic.log;
 
-import kcl.teamIndexZero.traffic.log.fileIO.FileInput;
+import kcl.teamIndexZero.traffic.log.FileIO.FileInput;
```

Comment:
All I see again is "that's bad" so really don't know what you are referring to. Will check it when online to understand what you mean.                                                                                                                                                                                                                                                                                                                                        Sent from my BlackBerry 10 smartphone on the EE network.                                                                                                                                                                                                                From: Alexander CherednichenkoSent: Monday 21 March 2016 08:11To: teamIndexZero/index_zero_trafficsystemReply To: teamIndexZero/index_zero_trafficsystemCc: kumar awijeetSubject: Re: [index_zero_trafficsystem] Gui graphics kumar awijeet (#9)In log/src/main/java/kcl/teamIndexZero/traffic/log/Log_Config.java:
> @@ -1,6 +1,6 @@
>  package kcl.teamIndexZero.traffic.log;
>  
> -import kcl.teamIndexZero.traffic.log.fileIO.FileInput;
> +import kcl.teamIndexZero.traffic.log.FileIO.FileInput;

that's bad.

—You are receiving this because you authored the thread.Reply to this email directly or view it on GitHub